backend/lib/services/infrastructureSecrets.js

```diff
@@ -266,8 +271,8 @@ exports.remove = async function ({ user, namespace, bindingName }) {
   }
 
   await Promise.all([
-    await Garden(user).namespaces(namespace).secretbindings.delete({ name: bindingName }),
-    await Core(user).namespaces(namespace).secrets.delete({ name: secretName })
+    await client['core.gardener.cloud'].secretbindings.delete({ namespace, name: bindingName }),
```

Comment:
introduce deleteCollection or ensure that delete is always called with a namespace and name
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#-strong-write-operations-pod-v1-core-strong-

Reply:
✔️